### ISSUE TITLE
Can't create ingress - Ingress.extensions "foo" is invalid:

### ISSUE THREAD

**Issue by bengray2** [rancher/rancher#35481]
**Rancher Server Setup**
- Rancher version: 2.6.2
- Installation option (Docker install/Helm Chart):  Docker install
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc):  RKE1
- Proxy/Cert Details:  self signed

**Information about the Cluster**
- Kubernetes version:  whatever default for rancher 2.6.2 is
- Cluster Type (Local/Downstream):
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider):

**Describe the bug**
I can't create an ingress.  No matter what value I use for "Name" I get the message:

`Ingress.extensions "example.com" is invalid: [spec.rules[0].http.paths[0].path: Invalid value: "": must be an absolute path, spec.tls[0].hosts[0]: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')]`

**To Reproduce**
Try to create an ingress and use any name.  Click create and the error message appears. 

**Result**
The error `Ingress.extensions "whatever" is invalid` and the ingress is not created.

**Expected Result**
Create the ingress with the name given.

**Screenshots**
With example.com (like the error message even gives as a valid example):
![image](https://user-images.githubusercontent.com/44791530/141430725-a2980e07-8f90-4738-8a3b-c98410d57ac0.png)

With 'something-else':
![image](https://user-images.githubusercontent.com/44791530/141430855-2dfe510a-1305-45fe-9b6c-64e005a5b01a.png)

With 'foo':
![image](https://user-images.githubusercontent.com/44791530/141430879-11f10b77-da2e-4036-9785-aca70b6a05af.png)

**Additional context**

**Comment by snasovich** [comment]
@bengray2 , the error doesn't seem to be related to ingress Name, but rather the fact that you need to fix other settings on it. For example, you want to fill out Host on "Certificates" tab you have on your screenshots - that's what the second error is about. The first one is likely due to not specifying "Path" on "Rules" tab.

Please update your values and let us know if you still run into this issue. If so, please provide screenshots of all tabs you see on Create Ingress page.

**Comment by bengray2** [comment]
@snasovich you are correct, thank you.  Is it possible to put UI validation on the form UI controls? When you first play with Rancher this isn't obvious at all, but as you develop your understanding of what is happening and the underlying YAML it's really obvious what I've done wrong now.

**Comment by snasovich** [comment]
@bengray2 , glad you were able to go past your initial issue. I'm going to transfer this issue to rancher/dashboard for UI team to consider as future enhancement.
